pkg/dev/amazonlinux2016.09/Dockerfile

```diff
@@ -0,0 +1,148 @@
+ && make deps \
+ && make \
+ && make strip
+USER root
+RUN cp -pr /home/"$OSQUERY_BUILD_USER"/osquery/build/linux/osquery/osqueryi /home/"$OSQUERY_BUILD_USER"/osquery/build/linux/osquery/osqueryd /opt/osquery \
+ && chown -R root. /opt/osquery \
+ && chmod -R 500 /opt/osquery/* \
+#put augeas lenses into the default directory that we changed earlier
+ && mkdir -p /opt/osquery/lenses \
+ && cp -r /usr/local/osquery/share/augeas/lenses/dist/* /opt/osquery/lenses \
+ && chmod -R 400 /opt/osquery/lenses/*
+RUN ls -lahR /opt/osquery/ && /opt/osquery/osqueryi --version
+
+#install packages that should be needed for ligbit2 compilation and successful pyinstaller run
+RUN yum -y install  \
+               python27-devel libffi-devel openssl-devel \
+               libxml2-devel libxslt-devel libjpeg-devel \
+               zlib-devel make cmake python27-setuptools \
+               gcc python-devel python-setuptools wget
+
+#libgit2 install start
+#must precede pyinstaller requirements
+ENV LIBGIT2_SRC_URL=https://github.com/libgit2/libgit2/archive/v0.26.0.tar.gz
+#it turns out github provided release files can change. so even though the code hopefully hasn't changed, the hash has.
+ENV LIBGIT2_SRC_SHA256=6a62393e0ceb37d02fe0d5707713f504e7acac9006ef33da1e88960bd78b6eac
+ENV LIBGIT2_SRC_VERSION=0.26.0
+ENV LIBGIT2TEMP=/tmp/libgit2temp
+RUN mkdir -p "$LIBGIT2TEMP" \
+ && cd "$LIBGIT2TEMP" \
+ && wget -q "$LIBGIT2_SRC_URL" -O libgit2.tar.gz \
+ && echo "$LIBGIT2_SRC_SHA256  libgit2.tar.gz" | sha256sum -c - \
+ && tar xzf libgit2.tar.gz \
+ && cd libgit2-"$LIBGIT2_SRC_VERSION"/ \
+ && cmake . -DCMAKE_INSTALL_PREFIX=/usr/local/ \
+ && make \
+ && make install
+
+#pyinstaller requirements start
+#must be preceded by libgit2 install
+COPY pyinstaller-requirements.txt /
+#default python-pip from yum does not like upgrading itself from pip. looking for better options other than wget.
+RUN wget -c https://bootstrap.pypa.io/get-pip.py \
+ && python get-pip.py \
+ && pip -v install -r pyinstaller-requirements.txt
+
+#fpm package making requirements start
+RUN yum install -y ruby ruby-devel rpmbuild rpm-build rubygems gcc make \
+ && gem install --no-ri --no-rdoc fpm
+
+#pyinstaller start
+#commands specified for ENTRYPOINT and CMD are executed when the container is run, not when the image is built
+#use the following variables to choose the version of hubble
+ENV HUBBLE_CHECKOUT=develop
+ENV HUBBLE_VERSION=2.2.10_develop
+ENV HUBBLE_GIT_URL=https://github.com/hubblestack/hubble.git
+ENV HUBBLE_SRC_PATH=/hubble_src
+ENV _HOOK_DIR="./pkg/"
+ENV _BINARY_LOG_LEVEL="INFO"
+ENV _INCLUDE_PATH=""
+ENV LD_LIBRARY_PATH=/lib:/lib64:/usr/lib:/usr/lib64:/usr/local/lib:/usr/local/lib64
+RUN git clone "$HUBBLE_GIT_URL" "$HUBBLE_SRC_PATH" \
+ && cd "$HUBBLE_SRC_PATH" \
+ && git checkout "$HUBBLE_CHECKOUT" \
+ && cp -rf "$HUBBLE_SRC_PATH" /hubble_build \
+ && echo __version__ = \"`git describe --tags | sed 's/v//'`\" > /hubble_build/hubblestack/__init__.py
```

Comment:
Would a string after version need to be in (single) quotes? Like
`version = '<string>'`

Reply:
Python accepts either. I used double quotes because it was easier to deal with on the shell.